transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpAcceptor.java

```diff
@@ -530,9 +559,19 @@ private void fireContentReceived(DefaultHttpSession session, HttpContentMessage
 
     @Override
     public void addBridgeFilters(IoFilterChain chain) {
-        IoSession transport = chain.getSession();
+        IoSession session = chain.getSession();
+
+        SocketAddress localAddress = session.getLocalAddress();
+
+        ResourceAddress transportAddress = LOCAL_ADDRESS.get(session);
+        ScheduledExecutorService taskExecutor = Executors.newScheduledThreadPool(1);
```

Comment:
I don't think we want to instantiate a ScheduledExecutorService for every session.  Instead get it via `@Resource` annotation for the ScheduleProvider